storage/conformance/conformance.go

```diff
@@ -98,6 +98,7 @@ func testAuthRequestCRUD(t *testing.T, s storage.Storage) {
 			Email:         "jane.doe@example.com",
 			EmailVerified: true,
 			Groups:        []string{"a", "b"},
+			Locale:        "en_US",
```

Comment:
Can we have test cases where we there's _no_ locale information stored, too?

Reply:
Removed locale in the secondary CRUD tests to verify that AuthRequest, AuthCode, and RefreshToken all function as intended.